app-sso/about.hbs.md

```diff
@@ -19,3 +18,37 @@ configured secure storage, and client restrictions.
 
 AppSSO's authorization server is based off of Spring Authorization Server project.
 For more information, see [Spring documentation](https://spring.io/projects/spring-authorization-server).
+
+## How this documentation is organized
+
+The AppSSO component documentation consists of the following sections that relate to what
+you are want to achieve:
```

Comment:
[suggestion]: I think this should read `you want to achieve:`

Reply:
ack ty